src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -176,3 +207,4 @@ def test_set_arg_const_scalar():
         otrans.apply(sched)
     assert ("Cannot generate OpenCL for Invokes that contain kernels with "
             "arguments passed by value" in str(err))
+    assert GOcean1p0OpenCLBuild(tmpdir).code_compiles(psy)
```

Comment:
You don't need this line as the point of this test is that we refuse to generation OpenCL code.